CHANGELOG_PENDING.md

```diff
@@ -9,6 +9,7 @@
 * Apps
 
 * Go API
+- [libs/autofile] Remove useless code in autofile package
```

Comment:
```suggestion
- [libs/autofile] \#3504 Remove useless code in autofile package (@guagualvcha)
```

Reply:
Would also be good to enumerate the removed public functions, namely: `Group#Search, Group#FindLast, GroupReader#ReadLine, GroupReader#PushLine`

Reply:
Very good point. I've updated the suggestion above.